.github/workflows/stale-issues.yaml

```
@@ -7,7 +7,7 @@ jobs:
     timeout-minutes: 1
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/stale@v9
+    - uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e # v9.0.0
```

`actions/stale@v9` used version `v9.1.0` in the last workflow run:
https://github.com/aquasecurity/trivy/actions/runs/17310893535/job/49144682743#step:1:40
https://github.com/actions/stale/releases/tag/v9.1.0

```suggestion
    - uses: actions/stale@5bef64f19d7facfb25b37b414482c7164d639639 # v9.0.0
```